Bricks/file/file.h

```diff
@@ -252,11 +256,11 @@ struct FileSystem {
   }
 
   template <typename F>
-  static inline void ScanDir(const std::string& directory, F&& f) {
+  static inline void ScanDir(const std::string& directory, F&& f, bool include_dirs = false) {
```

Comment:
Here too, `s/bool/ScanRecursive::Yes/`. (I can't think of a better name now. :-( ideally, it's the same symbol that `RmDir` uses, with a more neutral name.)